apps/auth/src/pages/index.tsx

```diff
@@ -20,9 +24,36 @@ function SignInOutButton() {
 
   return (
     <div className="flex flex-col gap-4">
+      <Checkbox
```

Comment:
auth frontend is EN only for now. not sure it is worth it to translate this at all..